integration_tests/commands/async/jsonresp_test.go

```diff
@@ -0,0 +1,56 @@
+package async
+
+import (
+	"testing"
+
+	testifyAssert "github.com/stretchr/testify/assert"
+	"gotest.tools/v3/assert"
+)
+
+func TestJSONRESP(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+	FireCommand(conn, "DEL key")
+
+	arrayAtRoot := `["dice",10,10.5,true,null]`
+	nestedArray := `{"b":["dice",10,10.5,true,null]}`
+
+	testCases := []struct {
+		name        string
+		commands    []string
+		expected    []interface{}
+		assert_type []string
+		jsonResp    []bool
+		nestedArray bool
+		path        string
+	}{
+		{
+			name:        "print array with mixed types",
+			commands:    []string{"json.set key $ " + arrayAtRoot, "json.resp key $"},
+			expected:    []interface{}{"OK", []interface{}{"[", "dice", int64(10), "10.5", "true", "(nil)"}},
+			assert_type: []string{"equal", "equal"},
+		},
+		{
+			name:        "print nested array with mixed types",
```

Comment:
Add a test case to run `JSON.RESP` at the root of a JSON with nested fields. I think it is already covered in the unit tests, let's add it here for completeness.